src/knx/bau57B0.cpp

```diff
@@ -9,16 +9,17 @@
 using namespace std;
 
 Bau57B0::Bau57B0(Platform& platform)
-    : BauSystemB(platform),
+    : BauSystemBDevice(platform),
       _ipParameters(_deviceObj, platform),
-      _dlLayer(_deviceObj, _addrTable, _ipParameters, _netLayer, _platform)
+      _dlLayer(_deviceObj, _ipParameters, _netLayer.getEntity(0), _platform)
 #ifdef USE_CEMI_SERVER
       ,
       _cemiServer(*this)
 #endif
 {
-    _netLayer.dataLinkLayer(_dlLayer);
+    _netLayer.getEntity(0).dataLinkLayer(_dlLayer);
```

Comment:
Just to nitpick: One could eliminate the magic number 0 ;)

Reply:
Same with all other getEntity calls. Are there more than 2 of them? If not getPrimary and getSecondary are easier to understand.

Reply:
The current KNX spec. does not forsee more than two interfaces, although they mention in the coupler model 2.0 spec that it could be an option in the future.
I will add public members getPrimary() and getSecondary().

Reply:
replaced.